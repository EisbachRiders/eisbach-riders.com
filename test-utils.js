import React from 'react'
import { render } from '@testing-library/react'
import { I18nextProvider } from 'react-i18next'
import i18n from './src/i18n/i18n'
import { ThemeProvider } from '@material-ui/styles'
import theme from './src/theme'

const AllTheProviders = ({ children }) => {
  return (
    <ThemeProvider theme={theme}>
      <I18nextProvider i18n={i18n}>{children}</I18nextProvider>
    </ThemeProvider>
  )
}

const customRender = (ui, options) =>
  render(ui, { wrapper: AllTheProviders, ...options })

// re-export everything
export * from '@testing-library/react'

// override render method
export { customRender as render }
