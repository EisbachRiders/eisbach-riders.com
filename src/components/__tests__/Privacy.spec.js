import React from 'react'
import { render } from '../../../test-utils'
import Privacy from '../Privacy'

describe('Privacy', () => {
  it('renders correctly', () => {
    const component = render(<Privacy />)
    expect(component).toMatchSnapshot()
  })
})
