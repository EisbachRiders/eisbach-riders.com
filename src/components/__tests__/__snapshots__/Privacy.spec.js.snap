// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Privacy renders correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="Hook-container-1eoh3z3"
        id="privacy"
      >
        <h6
          class="MuiTypography-root-1 MuiTypography-h6-11 Hook-title-1w1b37t"
        >
          privacy.privacy
        </h6>
        <h6
          class="MuiTypography-root-1 MuiTypography-subtitle1-12"
        >
          privacy.title1
        </h6>
        <p
          class="MuiTypography-root-1 MuiTypography-body2-2 Hook-text-kgq4fu"
        >
          privacy.terms1_1
        </p>
        <p
          class="MuiTypography-root-1 MuiTypography-body2-2 Hook-text-kgq4fu"
        >
          privacy.terms1_2
        </p>
        <p
          class="MuiTypography-root-1 MuiTypography-body2-2 Hook-text-kgq4fu"
        >
          privacy.terms1_3
        </p>
        <h6
          class="MuiTypography-root-1 MuiTypography-subtitle1-12"
        >
          privacy.title2
        </h6>
        <p
          class="MuiTypography-root-1 MuiTypography-body2-2 Hook-text-kgq4fu"
        >
          privacy.terms2_1
        </p>
        <ul
          class="MuiList-root-31 MuiList-padding-32"
        >
          <li
            class="MuiListItem-root-35 MuiListItem-default-38 MuiListItem-gutters-43 Hook-listItem-156lkrt"
          >
            <div
              class="MuiListItemText-root-47"
            >
              <span
                class="MuiTypography-root-1 MuiTypography-body1-3 MuiListItemText-primary-50 Hook-text-kgq4fu"
              >
                privacy.terms2_list1
              </span>
            </div>
          </li>
          <li
            class="MuiListItem-root-35 MuiListItem-default-38 MuiListItem-gutters-43 Hook-listItem-156lkrt"
          >
            <div
              class="MuiListItemText-root-47"
            >
              <span
                class="MuiTypography-root-1 MuiTypography-body1-3 MuiListItemText-primary-50 Hook-text-kgq4fu"
              >
                privacy.terms2_list2
              </span>
            </div>
          </li>
          <li
            class="MuiListItem-root-35 MuiListItem-default-38 MuiListItem-gutters-43 Hook-listItem-156lkrt"
          >
            <div
              class="MuiListItemText-root-47"
            >
              <span
                class="MuiTypography-root-1 MuiTypography-body1-3 MuiListItemText-primary-50 Hook-text-kgq4fu"
              >
                privacy.terms2_list3
              </span>
            </div>
          </li>
          <li
            class="MuiListItem-root-35 MuiListItem-default-38 MuiListItem-gutters-43 Hook-listItem-156lkrt"
          >
            <div
              class="MuiListItemText-root-47"
            >
              <span
                class="MuiTypography-root-1 MuiTypography-body1-3 MuiListItemText-primary-50 Hook-text-kgq4fu"
              >
                privacy.terms2_list4
              </span>
            </div>
          </li>
          <li
            class="MuiListItem-root-35 MuiListItem-default-38 MuiListItem-gutters-43 Hook-listItem-156lkrt"
          >
            <div
              class="MuiListItemText-root-47"
            >
              <span
                class="MuiTypography-root-1 MuiTypography-body1-3 MuiListItemText-primary-50 Hook-text-kgq4fu"
              >
                privacy.terms2_list5
              </span>
            </div>
          </li>
          <li
            class="MuiListItem-root-35 MuiListItem-default-38 MuiListItem-gutters-43 Hook-listItem-156lkrt"
          >
            <div
              class="MuiListItemText-root-47"
            >
              <span
                class="MuiTypography-root-1 MuiTypography-body1-3 MuiListItemText-primary-50 Hook-text-kgq4fu"
              >
                privacy.terms2_list6
              </span>
            </div>
          </li>
          <li
            class="MuiListItem-root-35 MuiListItem-default-38 MuiListItem-gutters-43 Hook-listItem-156lkrt"
          >
            <div
              class="MuiListItemText-root-47"
            >
              <span
                class="MuiTypography-root-1 MuiTypography-body1-3 MuiListItemText-primary-50 Hook-text-kgq4fu"
              >
                privacy.terms2_list7
              </span>
            </div>
          </li>
        </ul>
        <p
          class="MuiTypography-root-1 MuiTypography-body2-2 Hook-text-kgq4fu"
        >
          privacy.terms2_2
        </p>
        <h6
          class="MuiTypography-root-1 MuiTypography-subtitle1-12"
        >
          privacy.title3
        </h6>
        <p
          class="MuiTypography-root-1 MuiTypography-body2-2 Hook-text-kgq4fu"
        >
          privacy.terms3_1
        </p>
        <p
          class="MuiTypography-root-1 MuiTypography-body2-2 Hook-text-kgq4fu"
        >
          privacy.terms3_2
        </p>
        <p
          class="MuiTypography-root-1 MuiTypography-body2-2 Hook-text-kgq4fu"
        >
          privacy.terms3_3
        </p>
        <p
          class="MuiTypography-root-1 MuiTypography-body2-2 Hook-text-kgq4fu"
        >
          privacy.terms3_4
        </p>
        <ul
          class="MuiList-root-31 MuiList-padding-32"
        >
          <li
            class="MuiListItem-root-35 MuiListItem-default-38 MuiListItem-gutters-43 Hook-listItem-156lkrt"
          >
            <div
              class="MuiListItemText-root-47"
            >
              <span
                class="MuiTypography-root-1 MuiTypography-body1-3 MuiListItemText-primary-50 Hook-text-kgq4fu"
              >
                - Internet Explorer: https://support.microsoft.com/en-us/help/17442/windows-internet-explorer-delete-manage-cookies 
              </span>
            </div>
          </li>
          <li
            class="MuiListItem-root-35 MuiListItem-default-38 MuiListItem-gutters-43 Hook-listItem-156lkrt"
          >
            <div
              class="MuiListItemText-root-47"
            >
              <span
                class="MuiTypography-root-1 MuiTypography-body1-3 MuiListItemText-primary-50 Hook-text-kgq4fu"
              >
                - Firefox: https://www.mozilla.org/en-US/privacy/websites/#cookies 
              </span>
            </div>
          </li>
          <li
            class="MuiListItem-root-35 MuiListItem-default-38 MuiListItem-gutters-43 Hook-listItem-156lkrt"
          >
            <div
              class="MuiListItemText-root-47"
            >
              <span
                class="MuiTypography-root-1 MuiTypography-body1-3 MuiListItemText-primary-50 Hook-text-kgq4fu"
              >
                - Chrome: https://support.google.com/accounts/answer/61416?co=GENIE.Platform%3DDesktop&hl=en 
              </span>
            </div>
          </li>
          <li
            class="MuiListItem-root-35 MuiListItem-default-38 MuiListItem-gutters-43 Hook-listItem-156lkrt"
          >
            <div
              class="MuiListItemText-root-47"
            >
              <span
                class="MuiTypography-root-1 MuiTypography-body1-3 MuiListItemText-primary-50 Hook-text-kgq4fu"
              >
                - Safari: https://support.apple.com/en-gb/guide/safari/manage-cookies-and-website-data-sfri11471/mac 
              </span>
            </div>
          </li>
          <li
            class="MuiListItem-root-35 MuiListItem-default-38 MuiListItem-gutters-43 Hook-listItem-156lkrt"
          >
            <div
              class="MuiListItemText-root-47"
            >
              <span
                class="MuiTypography-root-1 MuiTypography-body1-3 MuiListItemText-primary-50 Hook-text-kgq4fu"
              >
                - Opera: https://help.opera.com/en/latest/web-preferences/#cookies 
              </span>
            </div>
          </li>
        </ul>
        <p
          class="MuiTypography-root-1 MuiTypography-body2-2 Hook-text-kgq4fu"
        >
          privacy.terms3_5
        </p>
        <h6
          class="MuiTypography-root-1 MuiTypography-subtitle1-12"
        >
          privacy.title4
        </h6>
        <p
          class="MuiTypography-root-1 MuiTypography-body2-2 Hook-text-kgq4fu"
        >
          privacy.terms4_1
        </p>
        <h6
          class="MuiTypography-root-1 MuiTypography-subtitle1-12"
        >
          privacy.title5
        </h6>
        <p
          class="MuiTypography-root-1 MuiTypography-body2-2 Hook-text-kgq4fu"
        >
          privacy.terms5_1
        </p>
        <h6
          class="MuiTypography-root-1 MuiTypography-subtitle1-12"
        >
          privacy.title6
        </h6>
        <p
          class="MuiTypography-root-1 MuiTypography-body2-2 Hook-text-kgq4fu"
        >
          privacy.terms6_1
        </p>
        <p
          class="MuiTypography-root-1 MuiTypography-body2-2 Hook-text-kgq4fu"
        >
          privacy.terms6_2
        </p>
        <h6
          class="MuiTypography-root-1 MuiTypography-subtitle1-12"
        >
          privacy.title7
        </h6>
        <p
          class="MuiTypography-root-1 MuiTypography-body2-2 Hook-text-kgq4fu"
        >
          privacy.terms7_1
        </p>
        <p
          class="MuiTypography-root-1 MuiTypography-body2-2 Hook-text-kgq4fu"
        >
          privacy.terms7_2
        </p>
        <p
          class="MuiTypography-root-1 MuiTypography-body2-2 Hook-text-kgq4fu"
        >
          privacy.terms7_3
        </p>
        <h6
          class="MuiTypography-root-1 MuiTypography-subtitle1-12"
        >
          privacy.title8
        </h6>
        <p
          class="MuiTypography-root-1 MuiTypography-body2-2 Hook-text-kgq4fu"
        >
          privacy.terms8_1
        </p>
        <a
          class="Hook-link-1by1ki0"
          href="javascript:gaOptout()"
        >
          privacy.terms8_3
        </a>
        <p
          class="MuiTypography-root-1 MuiTypography-body2-2 Hook-text-kgq4fu"
        >
          privacy.terms8_4
        </p>
        <h6
          class="MuiTypography-root-1 MuiTypography-subtitle1-12"
        >
          privacy.title9
        </h6>
        <p
          class="MuiTypography-root-1 MuiTypography-body2-2 Hook-text-kgq4fu"
        >
          privacy.terms9_1
        </p>
        <h6
          class="MuiTypography-root-1 MuiTypography-subtitle1-12"
        >
          privacy.title10
        </h6>
        <p
          class="MuiTypography-root-1 MuiTypography-body2-2 Hook-text-kgq4fu"
        >
          privacy.terms10_1
        </p>
        <ul
          class="MuiList-root-31 MuiList-padding-32"
        >
          <li
            class="MuiListItem-root-35 MuiListItem-default-38 MuiListItem-gutters-43 Hook-listItem-156lkrt"
          >
            <div
              class="MuiListItemText-root-47"
            >
              <span
                class="MuiTypography-root-1 MuiTypography-body1-3 MuiListItemText-primary-50 Hook-text-kgq4fu"
              >
                privacy.terms10_list1
              </span>
            </div>
          </li>
          <li
            class="MuiListItem-root-35 MuiListItem-default-38 MuiListItem-gutters-43 Hook-listItem-156lkrt"
          >
            <div
              class="MuiListItemText-root-47"
            >
              <span
                class="MuiTypography-root-1 MuiTypography-body1-3 MuiListItemText-primary-50 Hook-text-kgq4fu"
              >
                privacy.terms10_list2
              </span>
            </div>
             
          </li>
          <li
            class="MuiListItem-root-35 MuiListItem-default-38 MuiListItem-gutters-43 Hook-listItem-156lkrt"
          >
            <div
              class="MuiListItemText-root-47"
            >
              <span
                class="MuiTypography-root-1 MuiTypography-body1-3 MuiListItemText-primary-50 Hook-text-kgq4fu"
              >
                privacy.terms10_list3
              </span>
            </div>
             
          </li>
          <li
            class="MuiListItem-root-35 MuiListItem-default-38 MuiListItem-gutters-43 Hook-listItem-156lkrt"
          >
            <div
              class="MuiListItemText-root-47"
            >
              <span
                class="MuiTypography-root-1 MuiTypography-body1-3 MuiListItemText-primary-50 Hook-text-kgq4fu"
              >
                privacy.terms10_list4
              </span>
            </div>
             
          </li>
          <li
            class="MuiListItem-root-35 MuiListItem-default-38 MuiListItem-gutters-43 Hook-listItem-156lkrt"
          >
            <div
              class="MuiListItemText-root-47"
            >
              <span
                class="MuiTypography-root-1 MuiTypography-body1-3 MuiListItemText-primary-50 Hook-text-kgq4fu"
              >
                privacy.terms10_list5
              </span>
            </div>
             
          </li>
          <li
            class="MuiListItem-root-35 MuiListItem-default-38 MuiListItem-gutters-43 Hook-listItem-156lkrt"
          >
            <div
              class="MuiListItemText-root-47"
            >
              <span
                class="MuiTypography-root-1 MuiTypography-body1-3 MuiListItemText-primary-50 Hook-text-kgq4fu"
              >
                privacy.terms10_list6
              </span>
            </div>
             
          </li>
          <li
            class="MuiListItem-root-35 MuiListItem-default-38 MuiListItem-gutters-43 Hook-listItem-156lkrt"
          >
            <div
              class="MuiListItemText-root-47"
            >
              <span
                class="MuiTypography-root-1 MuiTypography-body1-3 MuiListItemText-primary-50 Hook-text-kgq4fu"
              >
                privacy.terms10_list7
              </span>
            </div>
             
          </li>
          <li
            class="MuiListItem-root-35 MuiListItem-default-38 MuiListItem-gutters-43 Hook-listItem-156lkrt"
          >
            <div
              class="MuiListItemText-root-47"
            >
              <span
                class="MuiTypography-root-1 MuiTypography-body1-3 MuiListItemText-primary-50 Hook-text-kgq4fu"
              >
                privacy.terms10_list8
              </span>
            </div>
             
          </li>
        </ul>
        <h6
          class="MuiTypography-root-1 MuiTypography-subtitle1-12"
        >
          privacy.title10_2
        </h6>
        <p
          class="MuiTypography-root-1 MuiTypography-body2-2 Hook-text-kgq4fu"
        >
          privacy.terms10_2
        </p>
        <h6
          class="MuiTypography-root-1 MuiTypography-subtitle1-12"
        >
          privacy.title11
        </h6>
        <p
          class="MuiTypography-root-1 MuiTypography-body2-2 Hook-text-kgq4fu"
        >
          privacy.terms11_1
        </p>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="Hook-container-1eoh3z3"
      id="privacy"
    >
      <h6
        class="MuiTypography-root-1 MuiTypography-h6-11 Hook-title-1w1b37t"
      >
        privacy.privacy
      </h6>
      <h6
        class="MuiTypography-root-1 MuiTypography-subtitle1-12"
      >
        privacy.title1
      </h6>
      <p
        class="MuiTypography-root-1 MuiTypography-body2-2 Hook-text-kgq4fu"
      >
        privacy.terms1_1
      </p>
      <p
        class="MuiTypography-root-1 MuiTypography-body2-2 Hook-text-kgq4fu"
      >
        privacy.terms1_2
      </p>
      <p
        class="MuiTypography-root-1 MuiTypography-body2-2 Hook-text-kgq4fu"
      >
        privacy.terms1_3
      </p>
      <h6
        class="MuiTypography-root-1 MuiTypography-subtitle1-12"
      >
        privacy.title2
      </h6>
      <p
        class="MuiTypography-root-1 MuiTypography-body2-2 Hook-text-kgq4fu"
      >
        privacy.terms2_1
      </p>
      <ul
        class="MuiList-root-31 MuiList-padding-32"
      >
        <li
          class="MuiListItem-root-35 MuiListItem-default-38 MuiListItem-gutters-43 Hook-listItem-156lkrt"
        >
          <div
            class="MuiListItemText-root-47"
          >
            <span
              class="MuiTypography-root-1 MuiTypography-body1-3 MuiListItemText-primary-50 Hook-text-kgq4fu"
            >
              privacy.terms2_list1
            </span>
          </div>
        </li>
        <li
          class="MuiListItem-root-35 MuiListItem-default-38 MuiListItem-gutters-43 Hook-listItem-156lkrt"
        >
          <div
            class="MuiListItemText-root-47"
          >
            <span
              class="MuiTypography-root-1 MuiTypography-body1-3 MuiListItemText-primary-50 Hook-text-kgq4fu"
            >
              privacy.terms2_list2
            </span>
          </div>
        </li>
        <li
          class="MuiListItem-root-35 MuiListItem-default-38 MuiListItem-gutters-43 Hook-listItem-156lkrt"
        >
          <div
            class="MuiListItemText-root-47"
          >
            <span
              class="MuiTypography-root-1 MuiTypography-body1-3 MuiListItemText-primary-50 Hook-text-kgq4fu"
            >
              privacy.terms2_list3
            </span>
          </div>
        </li>
        <li
          class="MuiListItem-root-35 MuiListItem-default-38 MuiListItem-gutters-43 Hook-listItem-156lkrt"
        >
          <div
            class="MuiListItemText-root-47"
          >
            <span
              class="MuiTypography-root-1 MuiTypography-body1-3 MuiListItemText-primary-50 Hook-text-kgq4fu"
            >
              privacy.terms2_list4
            </span>
          </div>
        </li>
        <li
          class="MuiListItem-root-35 MuiListItem-default-38 MuiListItem-gutters-43 Hook-listItem-156lkrt"
        >
          <div
            class="MuiListItemText-root-47"
          >
            <span
              class="MuiTypography-root-1 MuiTypography-body1-3 MuiListItemText-primary-50 Hook-text-kgq4fu"
            >
              privacy.terms2_list5
            </span>
          </div>
        </li>
        <li
          class="MuiListItem-root-35 MuiListItem-default-38 MuiListItem-gutters-43 Hook-listItem-156lkrt"
        >
          <div
            class="MuiListItemText-root-47"
          >
            <span
              class="MuiTypography-root-1 MuiTypography-body1-3 MuiListItemText-primary-50 Hook-text-kgq4fu"
            >
              privacy.terms2_list6
            </span>
          </div>
        </li>
        <li
          class="MuiListItem-root-35 MuiListItem-default-38 MuiListItem-gutters-43 Hook-listItem-156lkrt"
        >
          <div
            class="MuiListItemText-root-47"
          >
            <span
              class="MuiTypography-root-1 MuiTypography-body1-3 MuiListItemText-primary-50 Hook-text-kgq4fu"
            >
              privacy.terms2_list7
            </span>
          </div>
        </li>
      </ul>
      <p
        class="MuiTypography-root-1 MuiTypography-body2-2 Hook-text-kgq4fu"
      >
        privacy.terms2_2
      </p>
      <h6
        class="MuiTypography-root-1 MuiTypography-subtitle1-12"
      >
        privacy.title3
      </h6>
      <p
        class="MuiTypography-root-1 MuiTypography-body2-2 Hook-text-kgq4fu"
      >
        privacy.terms3_1
      </p>
      <p
        class="MuiTypography-root-1 MuiTypography-body2-2 Hook-text-kgq4fu"
      >
        privacy.terms3_2
      </p>
      <p
        class="MuiTypography-root-1 MuiTypography-body2-2 Hook-text-kgq4fu"
      >
        privacy.terms3_3
      </p>
      <p
        class="MuiTypography-root-1 MuiTypography-body2-2 Hook-text-kgq4fu"
      >
        privacy.terms3_4
      </p>
      <ul
        class="MuiList-root-31 MuiList-padding-32"
      >
        <li
          class="MuiListItem-root-35 MuiListItem-default-38 MuiListItem-gutters-43 Hook-listItem-156lkrt"
        >
          <div
            class="MuiListItemText-root-47"
          >
            <span
              class="MuiTypography-root-1 MuiTypography-body1-3 MuiListItemText-primary-50 Hook-text-kgq4fu"
            >
              - Internet Explorer: https://support.microsoft.com/en-us/help/17442/windows-internet-explorer-delete-manage-cookies 
            </span>
          </div>
        </li>
        <li
          class="MuiListItem-root-35 MuiListItem-default-38 MuiListItem-gutters-43 Hook-listItem-156lkrt"
        >
          <div
            class="MuiListItemText-root-47"
          >
            <span
              class="MuiTypography-root-1 MuiTypography-body1-3 MuiListItemText-primary-50 Hook-text-kgq4fu"
            >
              - Firefox: https://www.mozilla.org/en-US/privacy/websites/#cookies 
            </span>
          </div>
        </li>
        <li
          class="MuiListItem-root-35 MuiListItem-default-38 MuiListItem-gutters-43 Hook-listItem-156lkrt"
        >
          <div
            class="MuiListItemText-root-47"
          >
            <span
              class="MuiTypography-root-1 MuiTypography-body1-3 MuiListItemText-primary-50 Hook-text-kgq4fu"
            >
              - Chrome: https://support.google.com/accounts/answer/61416?co=GENIE.Platform%3DDesktop&hl=en 
            </span>
          </div>
        </li>
        <li
          class="MuiListItem-root-35 MuiListItem-default-38 MuiListItem-gutters-43 Hook-listItem-156lkrt"
        >
          <div
            class="MuiListItemText-root-47"
          >
            <span
              class="MuiTypography-root-1 MuiTypography-body1-3 MuiListItemText-primary-50 Hook-text-kgq4fu"
            >
              - Safari: https://support.apple.com/en-gb/guide/safari/manage-cookies-and-website-data-sfri11471/mac 
            </span>
          </div>
        </li>
        <li
          class="MuiListItem-root-35 MuiListItem-default-38 MuiListItem-gutters-43 Hook-listItem-156lkrt"
        >
          <div
            class="MuiListItemText-root-47"
          >
            <span
              class="MuiTypography-root-1 MuiTypography-body1-3 MuiListItemText-primary-50 Hook-text-kgq4fu"
            >
              - Opera: https://help.opera.com/en/latest/web-preferences/#cookies 
            </span>
          </div>
        </li>
      </ul>
      <p
        class="MuiTypography-root-1 MuiTypography-body2-2 Hook-text-kgq4fu"
      >
        privacy.terms3_5
      </p>
      <h6
        class="MuiTypography-root-1 MuiTypography-subtitle1-12"
      >
        privacy.title4
      </h6>
      <p
        class="MuiTypography-root-1 MuiTypography-body2-2 Hook-text-kgq4fu"
      >
        privacy.terms4_1
      </p>
      <h6
        class="MuiTypography-root-1 MuiTypography-subtitle1-12"
      >
        privacy.title5
      </h6>
      <p
        class="MuiTypography-root-1 MuiTypography-body2-2 Hook-text-kgq4fu"
      >
        privacy.terms5_1
      </p>
      <h6
        class="MuiTypography-root-1 MuiTypography-subtitle1-12"
      >
        privacy.title6
      </h6>
      <p
        class="MuiTypography-root-1 MuiTypography-body2-2 Hook-text-kgq4fu"
      >
        privacy.terms6_1
      </p>
      <p
        class="MuiTypography-root-1 MuiTypography-body2-2 Hook-text-kgq4fu"
      >
        privacy.terms6_2
      </p>
      <h6
        class="MuiTypography-root-1 MuiTypography-subtitle1-12"
      >
        privacy.title7
      </h6>
      <p
        class="MuiTypography-root-1 MuiTypography-body2-2 Hook-text-kgq4fu"
      >
        privacy.terms7_1
      </p>
      <p
        class="MuiTypography-root-1 MuiTypography-body2-2 Hook-text-kgq4fu"
      >
        privacy.terms7_2
      </p>
      <p
        class="MuiTypography-root-1 MuiTypography-body2-2 Hook-text-kgq4fu"
      >
        privacy.terms7_3
      </p>
      <h6
        class="MuiTypography-root-1 MuiTypography-subtitle1-12"
      >
        privacy.title8
      </h6>
      <p
        class="MuiTypography-root-1 MuiTypography-body2-2 Hook-text-kgq4fu"
      >
        privacy.terms8_1
      </p>
      <a
        class="Hook-link-1by1ki0"
        href="javascript:gaOptout()"
      >
        privacy.terms8_3
      </a>
      <p
        class="MuiTypography-root-1 MuiTypography-body2-2 Hook-text-kgq4fu"
      >
        privacy.terms8_4
      </p>
      <h6
        class="MuiTypography-root-1 MuiTypography-subtitle1-12"
      >
        privacy.title9
      </h6>
      <p
        class="MuiTypography-root-1 MuiTypography-body2-2 Hook-text-kgq4fu"
      >
        privacy.terms9_1
      </p>
      <h6
        class="MuiTypography-root-1 MuiTypography-subtitle1-12"
      >
        privacy.title10
      </h6>
      <p
        class="MuiTypography-root-1 MuiTypography-body2-2 Hook-text-kgq4fu"
      >
        privacy.terms10_1
      </p>
      <ul
        class="MuiList-root-31 MuiList-padding-32"
      >
        <li
          class="MuiListItem-root-35 MuiListItem-default-38 MuiListItem-gutters-43 Hook-listItem-156lkrt"
        >
          <div
            class="MuiListItemText-root-47"
          >
            <span
              class="MuiTypography-root-1 MuiTypography-body1-3 MuiListItemText-primary-50 Hook-text-kgq4fu"
            >
              privacy.terms10_list1
            </span>
          </div>
        </li>
        <li
          class="MuiListItem-root-35 MuiListItem-default-38 MuiListItem-gutters-43 Hook-listItem-156lkrt"
        >
          <div
            class="MuiListItemText-root-47"
          >
            <span
              class="MuiTypography-root-1 MuiTypography-body1-3 MuiListItemText-primary-50 Hook-text-kgq4fu"
            >
              privacy.terms10_list2
            </span>
          </div>
           
        </li>
        <li
          class="MuiListItem-root-35 MuiListItem-default-38 MuiListItem-gutters-43 Hook-listItem-156lkrt"
        >
          <div
            class="MuiListItemText-root-47"
          >
            <span
              class="MuiTypography-root-1 MuiTypography-body1-3 MuiListItemText-primary-50 Hook-text-kgq4fu"
            >
              privacy.terms10_list3
            </span>
          </div>
           
        </li>
        <li
          class="MuiListItem-root-35 MuiListItem-default-38 MuiListItem-gutters-43 Hook-listItem-156lkrt"
        >
          <div
            class="MuiListItemText-root-47"
          >
            <span
              class="MuiTypography-root-1 MuiTypography-body1-3 MuiListItemText-primary-50 Hook-text-kgq4fu"
            >
              privacy.terms10_list4
            </span>
          </div>
           
        </li>
        <li
          class="MuiListItem-root-35 MuiListItem-default-38 MuiListItem-gutters-43 Hook-listItem-156lkrt"
        >
          <div
            class="MuiListItemText-root-47"
          >
            <span
              class="MuiTypography-root-1 MuiTypography-body1-3 MuiListItemText-primary-50 Hook-text-kgq4fu"
            >
              privacy.terms10_list5
            </span>
          </div>
           
        </li>
        <li
          class="MuiListItem-root-35 MuiListItem-default-38 MuiListItem-gutters-43 Hook-listItem-156lkrt"
        >
          <div
            class="MuiListItemText-root-47"
          >
            <span
              class="MuiTypography-root-1 MuiTypography-body1-3 MuiListItemText-primary-50 Hook-text-kgq4fu"
            >
              privacy.terms10_list6
            </span>
          </div>
           
        </li>
        <li
          class="MuiListItem-root-35 MuiListItem-default-38 MuiListItem-gutters-43 Hook-listItem-156lkrt"
        >
          <div
            class="MuiListItemText-root-47"
          >
            <span
              class="MuiTypography-root-1 MuiTypography-body1-3 MuiListItemText-primary-50 Hook-text-kgq4fu"
            >
              privacy.terms10_list7
            </span>
          </div>
           
        </li>
        <li
          class="MuiListItem-root-35 MuiListItem-default-38 MuiListItem-gutters-43 Hook-listItem-156lkrt"
        >
          <div
            class="MuiListItemText-root-47"
          >
            <span
              class="MuiTypography-root-1 MuiTypography-body1-3 MuiListItemText-primary-50 Hook-text-kgq4fu"
            >
              privacy.terms10_list8
            </span>
          </div>
           
        </li>
      </ul>
      <h6
        class="MuiTypography-root-1 MuiTypography-subtitle1-12"
      >
        privacy.title10_2
      </h6>
      <p
        class="MuiTypography-root-1 MuiTypography-body2-2 Hook-text-kgq4fu"
      >
        privacy.terms10_2
      </p>
      <h6
        class="MuiTypography-root-1 MuiTypography-subtitle1-12"
      >
        privacy.title11
      </h6>
      <p
        class="MuiTypography-root-1 MuiTypography-body2-2 Hook-text-kgq4fu"
      >
        privacy.terms11_1
      </p>
    </div>
  </div>,
  "debug": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllBySelectText": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getAllByValue": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getBySelectText": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "getByValue": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllBySelectText": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryAllByValue": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryBySelectText": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "queryByValue": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
